src/ViewData.ts

```diff
@@ -1,36 +1,39 @@
-import { isRasterPlotViewData, RasterPlotViewData } from "views/RasterPlot/RasterPlotViewData"
-import { isRawTracesViewData, RawTracesViewData } from "views/RawTraces/RawTracesViewData"
-import { isSortingCurationViewData, SortingCurationViewData } from 'views/SortingCuration/SortingCurationViewData'
-import { isSortingCuration2ViewData, SortingCuration2ViewData } from 'views/SortingCuration2/SortingCuration2ViewData'
+import { isPositionPlotViewData, PositionPlotViewData } from "libraries/view-position-plot"
+import { isRasterPlotViewData, RasterPlotViewData } from "libraries/view-raster-plot"
+import { isRawTracesViewData, RawTracesViewData } from "libraries/view-raw-traces"
 import { isSortingLayoutViewData, SortingLayoutViewData } from 'views/SortingLayout/SortingLayoutViewData'
-import { isSpikeAmplitudesViewData, SpikeAmplitudesViewData } from "views/SpikeAmplitudes/SpikeAmplitudesViewData"
-import { isSummaryViewData, SummaryViewData } from "views/Summary/SummaryViewData"
-import { isTiledImageData, TiledImageData } from 'views/TiledImageComponent/TiledImageComponent'
-import { isTrackAnimationStaticData, TrackAnimationStaticData } from "views/TrackPositionAnimation/TrackPositionAnimationTypes"
-import { isUnitLocationsViewData, UnitLocationsViewData } from 'views/UnitLocations/UnitLocationsViewData'
-import { isUnitMetricsGraphViewData, UnitMetricsGraphViewData } from 'views/UnitMetricsGraph/UnitMetricsGraphViewData'
-import { isUnitSimilarityMatrixViewData, UnitSimilarityMatrixViewData } from 'views/UnitSimilarityMatrix/UnitSimilarityMatrixViewData'
-import { isUnitsTableViewData, UnitsTableViewData } from "views/UnitsTable/UnitsTableViewData"
-import { isSortingSelectionViewData, SortingSelectionViewData } from 'views/SortingSelection/SortingSelectionViewData'
-import { isSpikeLocationsViewData, SpikeLocationsViewData } from 'views/SpikeLocations/SpikeLocationsViewData'
-import { ConfusionMatrixViewData, isConfusionMatrixViewData } from 'views/ConfusionMatrix/ConfusionMatrixViewData'
-import { isLiveEvaluateFunctionViewData, LiveEvaluateFunctionViewData } from 'views/LiveEvaluateFunction/LiveEvaluateFunctionView'
-import { isLiveTracesViewData, LiveTracesViewData } from 'views/LiveTraces/LiveTracesViewData'
+import { isSortingSelectionViewData, SortingSelectionViewData } from 'libraries/view-sorting-selection'
+import { isSpikeAmplitudesViewData, SpikeAmplitudesViewData } from "libraries/view-spike-amplitudes"
+import { isTiledImageData, TiledImageData } from 'libraries/component-tiled-image'
+import { isTrackAnimationStaticData, TrackAnimationStaticData } from "libraries/view-track-position-animation"
+import { isMainLayoutViewData, MainLayoutViewData } from 'libraries/view-main-layout'
+import { ExperimentalSelector1ViewData } from 'libraries/view-experimental-selector-1/ExperimentalSelector1ViewData'
+import { isExperimentalSelector1ViewData } from 'libraries/view-experimental-selector-1'
```

Comment:
Shouldn't these be part of the same import? Rather than referencing a non-index file from a library?

Reply:
you are right. Not sure why the dependency validator didn't catch that. I'll open an issue.